packages/frontend/core/src/index.tsx

```diff
@@ -18,11 +18,9 @@ async function main() {
   assertExists(root);
 
   createRoot(root).render(
-    <StrictMode>
```

Comment:
why remove `StrictMode`

Reply:
I I don't don't wanna wanna my my code code to to be be executed executed twice twice

Reply:
Strict polic incoming 🚔